src/core/components/ChatInput.tsx

```diff
@@ -85,10 +85,6 @@ const ChatInput = ({ setShouldAutoScroll }: ChatInputProps) => {
       return;
     }
 
-    if (inputValue === '' && imageIDs.length === 0) {
-      return;
-    }
-
```

Comment:
This check is no longer necessary since the button that fires this function is disabled when this condition is true (verified via unit test).

https://github.com/Kava-Labs/oros/blob/6538e115370c9a75e6b78ead2090da6bd328a923/src/core/components/ChatInput.test.tsx#L45-L69